nixos/modules/services/admin/olivetin.nix

```diff
@@ -0,0 +1,62 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.olivetin;
+in
+{
+  options = {
```

Comment:
I find it a bit weird that there aren't any `settings` option with a yaml freeformat here. The upstream project seems to have a [bunch of settings that the user of the module might want to set](https://github.com/OliveTin/OliveTin#quickstart-reference-for-configyaml)